Dockerfile

```diff
@@ -4,24 +4,73 @@ FROM certbot/dns-route53:v1.32.0
 
 ARG VERSION
```

Comment:
I'm not sure if it's in scope but the `VERSION` build argument is not used in this image configuration.

Reply:
Please see commit ca8e4f0.